app/screens/Map.js

```diff
@@ -269,7 +268,7 @@ class Map extends Component {
                 </ConfirmationModal>
                 {isFetchingLocations ? <Text style={s.loading}>Loading...</Text> : null}
                 {maxZoom ? <Text style={s.loading}>Zoom in for updated results</Text> : null}
-                <View style={{ flex: 1, position: 'absolute', left: 0, top: -100, bottom: 0, right: 0 }}>
+                <View style={{ flex: 1, position: 'absolute', left: 0, top: 0, bottom: 0, right: 0 }}>
```

Comment:
@RyanTG, it would be great if we can come up with a more robust solution, but I can get the endless panning to stop if I add `top: 20` for my iPhone. It would be great if we could figure out a better solution, but it appears the map keeps trying to resize itself if the map is touching the phone system indicator stuff. There are ways to get the calculated header size, but when using `showHeader:false`, the header size appears to be 0, so I'm not certain we can use that. If we can find a way for the secondary header showing up to be hidden when using `headerTransparent:true`,  we might be ok. I will keep digging into this to try to find a better solution.

Reply:
There is a built-in solution, yay. If we replace the `View` with a `SafeAreaView`, the map is placed directly under our problem spots at the top. 🥳

Reply:
Dooooope